narwhals/_pandas_like/series.py

```diff
@@ -471,7 +471,12 @@ def abs(self) -> PandasLikeSeries:
     def cum_sum(self) -> PandasLikeSeries:
         return self._from_native_series(self._native_series.cumsum())
 
-    def unique(self) -> PandasLikeSeries:
+    def unique(self, *, maintain_order: bool = False) -> PandasLikeSeries:
+        """
+        NOTE:
+            The param `maintain_order` is only here for compatibility with the Polars API
+            and has no effect on the output.
+        """
```

Comment:
Maybe for pandas this is possible via `series.drop_duplicates(keep="first")`?

Reply:
pandas already maintains order, so I don't think we need to change anything

Reply:
Ignore me please 🙈